docs/code-rules/presentation/features.md

```diff
@@ -434,16 +434,26 @@ class FeatureNamePage extends StatelessWidget {
 
 ### Scaffolding
 
-//TODO
-
-#### A. AppBar
+Como se menciona en el artículo anterior, `BlocProvider` siempre **deben** retornar un `Scaffold`. Éste a su vez siempre **debe** contener en atributo `body`, una clase denominada `View`, que se **debe** crear siguiendo los lineamientos de su respectiva sección.
 
-//TODO
+Por último, el `Scaffold` tiene diferentes propiedades que **pueden** ser añadidas en este punto, como el `appBar` o los botones en la lista de widgets `persistentFooterButtons`.
```

Comment:
Podrias agregar una nota tipo info para que revisen la documentación oficial de Scaffold si quieren mas información sobre diferentes propiedades que pueden utilizar como esas que mencionas: 

:::info
Para más información acerca de las propiedades del widget `Scaffold`, revisar la [documentación oficial](https://api.flutter.dev/flutter/material/Scaffold-class.html).
:::

No necesariamente así, pero something like that jajaja